Formula/ksync.rb

```diff
@@ -28,7 +28,7 @@ def install
       -w
       -X #{project}/pkg/ksync.GitCommit=#{Utils.git_short_head}
       -X #{project}/pkg/ksync.GitTag=#{version}
-      -X #{project}/pkg/ksync.BuildDate=#{Utils.safe_popen_read("date", "+%Y%m%dT%H%M%S").chomp}
+      -X #{project}/pkg/ksync.BuildDate=#{Time.now.utc.rfc3339(9)}
```

Comment:
Why the `(9)`

Reply:
:shrug: I just took that from https://github.com/Homebrew/homebrew-core/pull/72669#discussion_r589164630

Reply:
9 decimal places to match what upstream uses

```
$ brew ruby -e 'puts Time.now.utc.rfc3339(9)'
2021-03-08T13:40:38.546814000Z
                    ^^^^^^^^^ 9 decimal places
```

Reply:
`9` means 9 decimal places of precision (so, nanoseconds).

Unnecessary? Probably. Is it what upstream does? Yes.

Reply:
I think it's padding?
```
[1] brew(main)> Time.now.rfc3339
=> "2021-03-08T13:41:32+00:00"
[2] brew(main)> Time.now.rfc3339(9)
=> "2021-03-08T13:41:35.392822000+00:00"
[3] brew(main)> Time.now.rfc3339(8)
=> "2021-03-08T13:41:41.43566000+00:00"
[4] brew(main)> Time.now.rfc3339(100)
=> "2021-03-08T13:41:44.6742580000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000+00:00"
```

Reply:
Okay